src/components/FileBrowser.js

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import _ from 'lodash';
-
-import '../vendor/prism-line-highlight.css';
```

Comment:
`PrimseLineHighlight` is used and should not be removed. It's used for giving a highlight of the code when you hover a feature. 

the file `prism-line-highlight.css` is actually custom code and not part of official prism line highlight plugin. Maybe not correct of me to put it in `vendor` folder. Anyway this file is needed and also `PrismLineHighlight` plugin code

Reply:
This is the highlight I'm talking about:
![image](https://user-images.githubusercontent.com/399339/67460855-5bcad180-f63c-11e9-9169-e6178db11a19.png)

Reply:
that's much better names! merging this in, thanks!